app/jobs/update_clinical_trials.rb

```diff
@@ -0,0 +1,28 @@
+class UpdateClinicalTrials < ActiveJob::Base
+  def perform(recurring = true)
+    begin
+      Source.where(source_type: 'PubMed').each do |source|
+        resp = Scrapers::PubMed.call_pubmed_api(source.citation_id)
+        clinical_trials = resp.clinical_trial_ids.uniq.map do |nct_id|
+          ClinicalTrial.where(nct_id: nct_id).first_or_create
+        end
+        source.clinical_trials = clinical_trials
+        source.save
+        sleep 0.1
```

Comment:
We may want a slightly longer sleep here unfortunately. On the plus side, since it happens in the background the slowness won't really matter.

Reply:
How long do you think would be best?

Reply:
Hrm, don't have a strong opinion. Maybe a quarter second or a half second?